capstone-frontend/src/components/MyClubs.jsx

```diff
@@ -28,7 +28,7 @@ class MyClubs extends Component {
           alert(err); 
         });
     
-    let allClubs = memberClubs.concat(ownerClubs.filter((item) => memberClubs.indexOf(item) < 0));
+    let allClubs = memberClubs.concat(ownerClubs.filter((item) => memberClubs.indexOf(item) < 0))
```

Comment:
nit: I think you may have accidentally deleted the semicolon here?

Reply:
I did! Thanks for catching that.